src/shared/models/setting/setting.controller.ts

```diff
@@ -14,6 +15,15 @@ import { SettingService } from './setting.service';
 export class SettingController {
   constructor(private readonly settingService: SettingService) {}
 
+  @Get('infoBanner')
+  @ApiBearerAuth()
+  @UseGuards(AuthGuard(), new RoleGuard(UserRole.ACCOUNT))
```

Comment:
I think we can omit authentication here -> remove `ApiBearerAuth` and `UseGuards`